src/main/java/org/javarosa/xpath/XPathConditional.java

```diff
@@ -90,7 +123,13 @@ public String evalReadable (DataInstance model, EvaluationContext evalContext) {
 
     public List<TreeReference> evalNodeset (DataInstance model, EvaluationContext evalContext) {
```

Comment:
@adamvert A List of TreeReferences is returned here which corresponds the the parent node of that queried state